index.html

```diff
@@ -8,41 +8,249 @@
+        <img
+          src="assets/Cellular Connection.png"
+          alt="icon cellular connection"
+        />
+        <img src="assets/Wifi.png" alt="icon wifi" />
+        <img src="assets/Battery.png" alt="icon battery" />
+      </div>
+    </section>
+    <header class="header">
+      <section class="container-header">
+        <a class="logo" href="#"> Henry Cadena </a>
+        <div class="burger-container">
+          <label class="burger"></label>
+          <label class="burger"></label>
+          <label class="burger"></label>
+        </div>
+      </section>
     </header>
-    <main class="container-main">
-      <!-- <img class="hero-image" src="assets/background-mobile-version.svg" alt="background"> -->
-      <div>
-        <h1>I'm Henry Cadena. Glad to see you!</h1>
+
+    <main class="main">
+      <section class="main-background-image">
+        <aside class="main-background-setup">
+          <h1>I'm Henry Cadena. Glad to see you!</h1>
+          <p>
+            I'm a software developer! I can help you build a product , feature
+            or website Look through some of my work and experience! If you like
+            what you see and have a project you need coded, don't hesitate to
+            contact me.
+          </p>
+          <h4>LET'S CONNECT</h4>
+          <article class="social-icons-container">
+            <a href="#"
+              ><img class="social-icons" src="assets/twitter.svg" alt="twitter"
+            /></a>
+            <a href="#"
+              ><img
+                class="social-icons"
+                src="assets/linkedin.svg"
+                alt="linkedin"
+            /></a>
+            <a href="#"
+              ><img class="social-icons" src="assets/m.svg" alt="m"
+            /></a>
+            <a href="#"
+              ><img class="social-icons" src="assets/github.svg" alt="github"
+            /></a>
+            <a href="#"
+              ><img class="social-icons" src="assets/peace.svg" alt="peace"
+            /></a>
+          </article>
+        </aside>
+      </section>
+    </main>
+
+    <section class="grid">
+      <aside>
+        <article class="work-card-container">
+          <img src="assets/Work1.png" alt="first project" />
+          <h2>Tonic</h2>
+          <div class="subtitle-card-container">
+            <h4>CANOPY</h4>
+            <ul class="ul-flex-features">
+              <li>Back End Dev</li>
+              <li>2015</li>
+            </ul>
+          </div>
+          <p>
+            A daily selection of privately personalized reads; no accounts or
+            sign-ups required
+          </p>
+          <div>
+            <ul class="ul-stack-flex">
+              <li class="li-style">HTML</li>
+              <li class="li-style">CSS</li>
+              <li class="li-style">JavaScript</li>
+            </ul>
+          </div>
+          <button class="btn-enable">See project</button>
+        </article>
+      </aside>
```

Comment:
Greeting!
You did great work implementing the work section, but there are a few changes that you still to work on to go to the next steps.

- Kindly, I suggest you follow the design as much as possible to make your project look more professional.

> ## The Figma Design
>
><img width="314" alt="Screenshot 2023-06-07 at 4 36 46 PM" src="https://github.com/henrycadenaec/PortfolioFinishMobileVersion/assets/91237525/596eae2e-572c-4a4e-b8a7-cbed457c6ae0">
>
> ## Your Design
>
><img width="375" alt="Screenshot 2023-06-07 at 4 37 44 PM" src="https://github.com/henrycadenaec/PortfolioFinishMobileVersion/assets/91237525/49b7ff98-0236-4536-9f79-6b937f218231">

Reply:
Thanks @stanleySimeon